operators/src/source/gdal_source.rs

```diff
@@ -254,6 +254,11 @@ where
         let rasterband: GdalRasterBand =
             dataset.rasterband(data_set_params.rasterband_channel as isize)?;
 
+        // get the no_data_value. TODO: move to dataset parameters? Keep in mind, that this can change for each time step.
+        // We also need to ge metadata from the dataset (for each tile) e.g. scale + offset.
+        let no_data_value = rasterband.no_data_value().map(T::from_);
+        let fill_value = no_data_value.unwrap_or_else(T::zero);
```

Comment:
I think we should enforce a no data value. cf. my other comment

Reply:
yes but i dno'T know how to do this at the moment.

Reply:
yeah, just create a New Idea task for it.

Reply:
ok

Reply:
and link it here so I can resolve the discussion ;)

Reply:
https://trello.com/c/pjchxCTQ